Description: Ignore tests which run cargo
 If we run this test at build time, the cargo call will generate a Cargo.lock
 file, which gets installed to the package.  That then causes autopkgtest to
 fail when any dependency versions are updated.
 .
 We can't run the build time test and then exclude installation of Cargo.lock,
 because autopkgtest will run the test and fail trying to write Cargo.lock to a
 write protected directory.
 .
 Instead, just skip running the test.
Author: James McCoy <jamessan@debian.org>
Forwarded: not-needed
---
--- a/tests/piped.rs
+++ b/tests/piped.rs
@@ -4,6 +4,7 @@ use std::io::Write;
 use std::process::{Command, Stdio};
 
 #[test]
+#[ignore]
 fn piped_password() {
     // Run an example that reads a password and prints it
     let mut out = Command::new("cargo")
